core/jazz_t-vault/components/utils/vault.js

```diff
@@ -401,7 +458,9 @@ module.exports = {
   createUserInSafe,
   deleteUserFromSafe,
   getRoleInSafe,
+  createRole,
```

Comment:
createRole combines createRoleInVault and createRoleInSafe?
Also, does it check to see if role already exists in vault and safe before creating?

similarly we need createUser which combines createUserInVault and createUserInSafe.
Again it should check to see if user already exists in vault and safe before creating it.

Reply:
Done

Reply:
This will be addressed in a separate story.